docs/GDD/OPENCOLD.md

```diff
@@ -119,17 +119,22 @@ Pretend that your were pitching your game to a executive going to the elevator.
   - <Reason #4>
   - <etc.>
 
-## Core Gameplay Mechanics (Detailed)
-  - [Core Gameplay Mechanic #1]
-    - [Details] /Describe in 2 Paragraphs or less/
+##Core Gameplay Mechanics (Detailed)
+  - Combat
+    - [Details] 
     - [How it works] /Describe in 2 Paragraphs or less/
-  - [Core Gameplay Mechanic #2]
-    - [Details] /Describe in 2 Paragraphs or less/
+  - Exploration
+    - [Details] 
     - [How it works] /Describe in 2 Paragraphs or less/
-  - [Core Gameplay Mechanic #3]
-    - [Details] /Describe in 2 Paragraphs or less/
+  - Resource Management
```

Comment:
1. I feel a clearer divide between what an inventory resource and what a general resource is wouldn't be a bad idea. 
2. The order should also probably be shifted around (Health -> Heat -> Heat Producers -> Weapons and related -> Implants).
3. You repeated implants twice :P

Reply:
1. Done.
2. Done.
3. Oops. Done.